matrix_client/api.py

```diff
@@ -294,6 +294,94 @@ def set_room_topic(self, room_id, topic):
         }
         return self._send("PUT", "/rooms/" + room_id + "/state/m.room.topic", body)
 
+    def get_power_levels(self, room_id):
+        """Perform GET /rooms/$room_id/state/m.room.power_levels
+        Args:
+            room_id(str): The room ID
+        """
+        return self._send("GET", "/rooms/" + room_id + "/state/m.room.power_levels")
+
+    def modify_user_power_levels(self, room_id, users=None, users_default=None):
```

Comment:
Is there any way you could possibly merge these two functions. There is some overlap which I'd rather not have if it can be avoided. Perhaps a ``set_power_levels`` function which just has ``room_id`` and ``**kargs`` as arguments.